www/src/views/builder/index.ejs

```diff
@@ -351,11 +364,37 @@
 												</td>
 											</tr>
 										</tbody>
+										<tbody ng-if="isRuneCrafting">
+											<td scope="row"
+												<span> <!-- charm selector-->
+													<select class="custom-select" ng-model="charm1Selector" ng-change="runeCraftCharms($event)">
+														<option ng-value="'A'">--Select--</option>																			
```

Comment:
Nice job picking up on that "'A'" for the `ng-value`. Those can be tricky and it can be very easy to set it to "A" instead of "'A'". Good find

Reply:
Totally not necessary, but if you're looking for a challenge, you could get even more clever with this and put the `charm#Selector`s into an array and `ng-repeat` over those as well. The javascript would have to update to account for that of course, but it could be interesting.

Reply:
ohh, that sounds fun. I'll try that in the next PR.